core/shared/src/main/scala/zio/Clock.scala

```diff
@@ -271,6 +275,20 @@ object Clock extends ClockPlatformSpecific with Serializable {
     def scheduler(implicit trace: ZTraceElement): UIO[Scheduler] =
       ZIO.succeed(globalScheduler)
 
+    def javaClock(implicit trace: ZTraceElement): UIO[java.time.Clock] = {
```

Comment:
We may have a slight bias here for `asJava` or similar (`asJavaClock`?).

e.g.:

 - `asScheduledExecutorService`
 - `asDuration`

Reply:
Yes I am wondering if it should be `toJava` or `toJavaClock` since it returns an effect.

Reply:
It is a little weird when used as an accessor (`Clock.asJava` or `Clock.toJava`). Maybe a reason to leave as is. Or maybe we delete the accessor.

Reply:
Good point! Maybe we leave as-is 👍